docs/tutorial/errors/README.md

```diff
@@ -0,0 +1,129 @@
+---
+order: 6
+---
+
+# Errors and Retries
+
+## Errors
+
+When an error occurs during a task, your flow will fail. It can be interesting to execute some actions before terminating the flow, for example, sending mail to your team.
```

Comment:
```suggestion
When an error occurs during a task, your flow will fail. It can be interesting to execute some actions before terminating the flow, for example, sending an email to your team.
```